speechbrain/lobes/models/huggingface_transformers/mimi.py

```diff
@@ -0,0 +1,174 @@
+"""This lobe enables the integration of huggingface pretrained Mimi.
+
+Mimi codec is a state-of-the-art audio neural codec, developed by Kyutai, that combines semantic and acoustic information into audio tokens running at 12Hz and a bitrate of 1.1kbps.
+
+Note that you need to install transformers>=4.45.1` to use this module.
+
+Repository: https://huggingface.co/kyutai/mimi
+Paper: https://kyutai.org/Moshi.pdf
+
+Authors
+ * Pooneh Mousavi 2024
+"""
+
+import torch
+
+from speechbrain.dataio.dataio import clean_padding_, length_to_mask
+from speechbrain.lobes.models.huggingface_transformers.huggingface import (
+    HFTransformersInterface,
+)
+from speechbrain.utils.logger import get_logger
+
+logger = get_logger(__name__)
+
+
+class Mimi(HFTransformersInterface):
+    """An wrapper for the HuggingFace Mimi model
+
+    Arguments
+    ---------
+    source : str
+        A HuggingFace repository identifier or a path
+    save_path : str
+        The location where the pretrained model will be saved
+    sample_rate : int (default: 24000)
+        The audio sampling rate
+    freeze : bool
+        whether the model will be frozen (e.g. not trainable if used
+        as part of training another model)
+    num_codebooks : int  (default: 8)
+        Number of qunatizer. It could be [2,3,4,5,6,7,8]
```

Comment:
Typo @pplantinga test failing to catch it? :'(

Reply:
what is the typo here? double space?

Reply:
Dunno what a qunatizer is.

Reply:
I guess the spell checker doesn't know how to spell `quantizer`